app/controllers/micropost_controller.rb

```diff
@@ -0,0 +1,55 @@
+class MicropostController < ApplicationController
```

Comment:
mô tả dòng code này thử e

Reply:
nhớ kĩ mô hình MVC trong chapter này luôn nhé, reply vào đây luôn e

Reply:
controller MicropostController kế thừa lại các tính năng của class ApplicationController

MVC bao gồm (Model, View, Controller)
Model: dùng để tương tác với db và query, lưu dữ liệu
View: dùng để nhận dữ liệu và hiển thị giao diện cho người dùng
Controller: dùng để xử lý các yêu cầu của người dùng, tương tác với Model lấy data trả về View để hiển thị

Reply:
chưa ổn lắm e. a nhớ trong chapter 2 nó có vẽ cái mô hình MVC đó, e có xem k?

Reply:
dạ e xem rồi a
e trả lời lại ạ

- người dùng gửi request đến BE
- router nhận request và ánh xạ đến action trong Controller
- Controller tương tác với Model 
- Model truy vấn lấy dữ liệu và trả lại cho Controller 
- Controller gửi dữ liệu đến View và View hiển thị giao diện cho người dùng